core/src/services/azblob/backend.rs

```diff
@@ -545,6 +545,7 @@ impl Access for AzblobBackend {
                 presign_stat: self.has_sas_token,
                 presign_read: self.has_sas_token,
                 presign_write: self.has_sas_token,
+                presign_delete: self.has_sas_token,
```

Comment:
If you don't intend to implement `presign_delete` support for `azblob` in this PR, please leave this field unchanged so our tests will skip testing on these services.

This comment applies to all services.